### ISSUE TITLE
Feature request: Shortened display names of the sessions

### ISSUE THREAD

**Issue by knmac** [Shatur/neovim-session-manager#27]
**Describe the problem or limitation you are having**

Sometimes the path to a session is too long, so it is hard to distinguish the paths that are only different at the very end, e.g.,
```
/the/path/to/the/session/number/one
/the/path/to/the/session/number/two
```
The extra characters of each path are removed, so they are displayed (via Telescope) as
```
/the/path/to/the/session/num
/the/path/to/the/session/num
```

**Describe the solution you'd like**

It would be great if the paths are shortened if they are longer than the width of the Telescope's window. The expected outcome is something like this:
```
/t/p/t/t/session/number/one
/t/p/t/t/session/number/two
```

It looks like this can be changed in the file `init.lua`, lines 15-17, and lines 61-63. For example, instead of
```lua
  for _, session in ipairs(sessions) do
    table.insert(display_names, session.dir.filename)
  end
```
we can have something like this:
```lua
  for _, session in ipairs(sessions) do
    table.insert(display_names, shorten_path(session.dir.filename))
  end
```

**Describe alternatives you've considered**

Instead of looking at the width of Telescope's window, we can also add a variable to set the maximum length of the path. This can be changed in the config.

**Comment by Shatur** [comment]
Sounds like a reasonable idea. I wonder if there is something for this in Plenary.

> we can also add a variable to set the maximum length of the path. This can be changed in the config.

I would go with this since we do not depend on Telescope.

I'm a little busy right now, but PR is welcome.

**Comment by Shatur** [comment]
Closed by #30.
